xAPI.md

```diff
@@ -2331,30 +2350,31 @@ of "http://adlnet.gov/expapi/attachments/signature" and a contentType of "applic
 X.509 certificate.
 * If X.509 was used to sign, the JWS header SHOULD include the "x5c" property containing
 the associated certificate chain.
+
+##### LRS Requirements
 * The LRS MUST reject requests to store Statements that contain malformed signatures, with HTTP 400.
-* The LRS SHOULD include a message in the response of a rejected statement.  
+* The LRS SHOULD include a message in the response of a rejected statement. 
 * In order to verify signatures are well formed, the LRS MUST do the following:
     * Decode the JWS signature, and load the signed serialization of the Statement from the
       JWS signature payload.
-    * Validate that the "original" Statement is logically equivalent to the received Statement.
-        * When making this equivalence check, differences which could have been caused by
-        allowed or required LRS processing of "id", "authority", "stored", "timestamp", or
```

Comment:
This is now covered in the Statement comparison requirements section. I checked that I didn't miss any of the properties but please double check as this is completely re-written and done in a totally different way.